src/env-inl.h

```diff
@@ -303,7 +303,17 @@ inline Environment::Environment(IsolateData* isolate_data,
   v8::HandleScope handle_scope(isolate());
   v8::Context::Scope context_scope(context);
   set_as_external(v8::External::New(isolate(), this));
-  set_binding_cache_object(v8::Object::New(isolate()));
+
+  v8::Local<v8::Primitive> Null = v8::Null(isolate());
```

Comment:
Style: `null`, `Null` makes it look like a type.

Reply:
done